freshbooks/models.py

```diff
@@ -141,6 +142,19 @@ def __str__(self) -> str:
     def __repr__(self) -> str:  # pragma: no cover
         return "ListResult({})".format(self._name)
 
+    def __add__(self, other: "ListResult") -> "ListResult":
+        if not isinstance(other, ListResult) or (self._name != other._name):
+            raise TypeError("Objects not of same ListResult type")
+
+        data = self.data
+        data[self._name] = self.data.get(self._name, []) + other.data.get(self._name)
```

Comment:
Could we not have used list1.extend(list2) here, I am guessing it not as clean with the self._name?

Reply:
Thanks! This made me realize that I was mutating the original object. 
Fixed that and now using `extend`.